### DISCUSSION TITLE
Adding Navigation Indicator (bullets) only when 2 or more items

### DISCUSSION THREAD

**Discussion by enzioardesch** [dimsemenov/PhotoSwipe#1909] [Q&A]
I want to implement the navigation indicator bullets, but with the current provided example [(https://photoswipe.com/adding-ui-elements/#adding-navigation-indicator-bullets)](https://photoswipe.com/adding-ui-elements/#adding-navigation-indicator-bullets) it shows a bullet if there is only one element. That isn't necessary for obvious reasons. Is there a way to edit this so that the bullets only become active if there are 2 or more elements?

I am currently using the code as provided in the example.

**Comment by goldipl** [comment]
I think the best option is to write JS for it and hide bulllet with simple css
